tests/QARegressionTests/test_Issues/test_CoreIssue328.py

```diff
@@ -0,0 +1,49 @@
+from cdisc_rules_engine.config.config import ConfigService
+from cdisc_rules_engine.operations.valid_codelist_dates import ValidCodelistDates
+from cdisc_rules_engine.models.operation_params import OperationParams
+import pandas as pd
+from cdisc_rules_engine.services.cache.cache_service_factory import CacheServiceFactory
+from cdisc_rules_engine.constants.cache_constants import PUBLISHED_CT_PACKAGES
+import pytest
+
+"""These tests have been taken from unit tests written by nhyadel.

```

Comment:
I don't think we need to duplicate unit tests in regression testing

Reply:
removed